x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/integration_tests/response_actions_log.test.tsx

```diff
@@ -1537,8 +1538,8 @@ describe('Response actions history', () => {
       );
     });
 
-    it('should show a list of actions (with `scan`) when opened', async () => {
-      render();
+    it('should show a list of actions (with `runscript`) when opened', async () => {
+      render({ 'data-test-height': 350 });
```

Comment:
Just a note that we would need to have larger heights here as the number of commands increases. We might want to add a note somewhere here about that.